README.rst

```diff
@@ -53,6 +53,10 @@ For detailed instructions on how to install NAV, see
 `<doc/intro/install.rst>`_. See the file `<NOTES.rst>`_ for release notes and
 upgrade instructions.
 
+Take it for a quick spin
+------------------------
+For instruction on how to use docker to run code straight from the repo, see
+`<doc/hacking/using-docker.rst>`_.
```

Comment:
If you're going to reference this directly from the README, I would add something like this:

```suggestion
`<doc/hacking/using-docker.rst>`_.  Please be aware, though, that this is a highly developer-oriented environment, and is not suitable for production use. 
```

Also, maybe potentially point to https://github.com/Uninett/nav-docker as a production-oriented alternative.